core/include/optee_rpc_cmd.h

```diff
@@ -148,6 +148,18 @@
  */
 #define OPTEE_RPC_CMD_BENCH_REG		20
 
+/*
+ * Transfer data in/out via I2C
+ *
+ * [in]     value[0].a	    Transfer mode (read/write).
```

Comment:
s/(read/write)/(write = 0, read = 1)/

Reply:
why 0 and 1? and how is this different from the statement just above (copied here for you to compare).
unless you want me to also send a patch to fix OPTEE_RPC_CMD_BENCH_REG?

/*
 * Register timestamp buffer in the linux kernel optee driver
 *
 * [in]     value[0].a	    Subcommand (register buffer, unregister buffer)
 * [in]     value[0].b	    Physical address of timestamp buffer
 * [in]     value[0].c	    Size of buffer
 */
#define OPTEE_RPC_CMD_BENCH_REG		2

Reply:
This header file should be the reference for the API. It should better describe it explicitly.

For example, you could go this way:
```c
/*
 * Transfer data in/out via I2C
 *
 * [in]     value[0].a	    Transfer mode: enum rpc_i2c_mode.
 * (...)
 */
#define OPTEE_RPC_CMD_I2C_TRANSFER	21

enum rpc_i2c_mode {
    RPC_I2C_MODE_WRITE = 0,
    RPC_I2C_MODE_READ = 1,
};